### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by tundeiness**
Removed the import of the style.css on the index.html, and comments in other javascript files

### CODE REVIEW

File: src/contact.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  para.innerHTML = `${paraContent} <small class="text-muted"> 091 235 4329 & 213 453 765</small>`;
+
+  inner.appendChild(para);
+
+  const horizontal = document.createElement('hr');
+  horizontal.setAttribute('class', 'my-4');
+
+  inner.appendChild(horizontal);
+
+  // socials
```
Comment (azdrenymeri) [inline src/contact.js]: kindly remove the comment

File: src/menu.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const menuTab = (() => {
+  const inner = document.createElement('div');
+  inner.setAttribute('class', 'inner d-flex flex-row justify-content-around pt-4');
+
+  // card container
```
Comment (azdrenymeri) [inline src/menu.js]: Also here remove the comment.

File: src/menu.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    img: 'https://user-images.githubusercontent.com/25479050/69044190-c4fff380-09f4-11ea-9741-5eda04e49a12.jpeg', title: 'Egusi Soup', price: '$35', text: 'This is a longer card with supporting text below as a natural lead-in to additional content.',
+  },
+  {
+    img: 'https://user-images.githubusercontent.com/25479050/69044258-e82aa300-09f4-11ea-9a8e-8e3fe185e347.jpeg', title: 'Yam Porridge', price: '$50', text: 'This is a longer card with supporting text below as a natural lead-in to additional content.',
+  },
+  {
+    img: 'https://user-images.githubusercontent.com/25479050/69044322-07293500-09f5-11ea-9133-f27b6c441705.jpeg', title: 'Vegetable Soup', price: '$120', text: 'This is a longer card with supporting text below as a natural lead-in to additional content.',
+  }];
+
+  // first menu
```
Comment (azdrenymeri) [inline src/menu.js]: And here

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
### Hello @tundeiness 👋 
#### Nice job on this project 👍 
#### There is this error 
*  Kindly remove the import of the `style.css` on the `index.html`.
   The `style.css` is already imported on your `index.js` `line 5: ( import './style.css'; )`:
`Failed to load resource: net::ERR_FILE_NOT_FOUND /C:/src/style.css:1 `
[Learn about Webpack loaders](https://medium.com/a-beginners-guide-for-webpack-2/webpack-loaders-css-and-sass-2cc0079b5b3a)
scr

### kindly fix this issue and add it for a re-review

**Review by shloch — APPROVED** [review]
We are done here, nicely executed project

Great job @tundeiness  Omotunde

### You can merge this Request 😊😊

**Pull request merged**